bot/main.py

```diff
@@ -0,0 +1,292 @@
+def clear_command(message: types.Message) -> None:
+    """Remove info about active polls related to User from active_polls.json.
+
+    Args:
+        message: Message from User with /clear command in it.
+
+    """
+    logger.info(
+        f"\"{message.text}\" command from user {message.from_user.username} (id {message.from_user.id}) has been received."
+    )
+    active_polls = active_polls_checker(message.chat.id)
+    if active_polls:
+        for poll in active_polls:
+            JSONHandler.dict_pop("active_polls.json", poll)
+            logger.info(f"Poll {poll} has been removed from active_polls.json.")
+    bot.send_message(message.chat.id, botmessages.CLEAR_CHAT_HISTORY)
+    logger.info(f"Active polls have been cleared for user {message.from_user.id} (id {message.chat.id}).")
+
+
+@bot.message_handler(commands=["clear_my_score"])
+def clear_my_score_command(message: types.Message) -> None:
+    """Remove score entry of User from scores.json.
+
+    Args:
+        message: Message from User with /clear_my_score command in it.
+
+    """
+    logger.info(
+        f"\"{message.text}\" command from user {message.from_user.username} (id {message.from_user.id}) has been received."
+    )
+    if str(message.from_user.id) in list(JSONHandler.read("scores.json"))[:]:
+        JSONHandler.dict_pop("scores.json", str(message.from_user.id))
+    bot.send_message(message.chat.id, botmessages.CLEAR_SCORE)
+    logger.info(f"Score record for user {message.from_user.username} (id {message.from_user.id}) has been removed.")
+
+
+@bot.poll_answer_handler()
+def handle_poll_answer(poll_answer: types.PollAnswer) -> None:
+    """Update scores.json file with new scores and send new poll to the user if there is no active one; or forward
+    existing active poll to User.
+
+    Args:
+        poll_answer: Poll answer from user.
+
+    """
+    # Providing article link to User.
+    doc_link = JSONHandler.read("active_polls.json")[str(poll_answer.poll_id)]["doc_link"]
+    if doc_link:
+        logger.info(f"Documentation link is being provided to User.")
+        markup = telebot.types.InlineKeyboardMarkup(row_width=1)
+        article = types.InlineKeyboardButton(text="Article", url=doc_link)
+        markup.add(article)
+        bot.edit_message_reply_markup(
+            chat_id=poll_answer.user.id,
+            message_id=JSONHandler.read("active_polls.json")[str(poll_answer.poll_id)]["message_id"],
+            reply_markup=markup
+        )
+        logger.info(f"Documentation link has been provided to User.")
+
+    logger.info(f"User {poll_answer.user.username} (id {poll_answer.user.id}) answered {poll_answer.option_ids[0]}.")
+    initial_user_score = {
+        str(poll_answer.user.id): {
+            "correct": 0,
+            "total": 0
+        }
+    }
+
+    # Checking weather the entry about the user is absent in scores.json and make a new entry if so.
+    scores: Dict[str, Dict[str, int]] = JSONHandler.read("scores.json")
```

Comment:
```suggestion
    scores = JSONHandler.read("scores.json")
```
в read уже есть тайп хинт